stubs/docutils/docutils/nodes.pyi

```diff
@@ -129,19 +139,76 @@ class Element(Node):
     def __add__(self, other: list[Node]) -> list[Node]: ...
     def __radd__(self, other: list[Node]) -> list[Node]: ...
     def __iadd__(self, other: Node | Iterable[Node]) -> Self: ...
-    def copy(self) -> Self: ...
-    def deepcopy(self) -> Self: ...
-    def pformat(self, indent: str = "    ", level: int = 0) -> str: ...
     def astext(self) -> str: ...
-    def index(self, item: Node, start: int = 0, stop: int = sys.maxsize) -> int: ...
-    def remove(self, item: Node) -> None: ...
+    def non_default_attributes(self) -> dict[str, Any]: ...
+    def attlist(self) -> list[tuple[str, Any]]: ...
+    @overload
+    def get(self, key: str) -> Any: ...
+    @overload
+    def get(self, key: str, failobj: _T) -> _T: ...
+    def hasattr(self, attr: str) -> bool: ...
+    def delattr(self, attr: str) -> None: ...
+    @overload
+    def setdefault(self, key: str) -> Any: ...
+    @overload
+    def setdefault(self, key: str, failobj: _T) -> _T: ...
```

Comment:
```suggestion
    def setdefault(self, key: str, failobj: _T) -> Any | _T: ...
```

Reply:
if i were to add that suggestion i would have:

```python
   @overload
    def setdefault(self, key: str) -> Any: ...
    @overload
    def setdefault(self, key: str, failobj: _T) -> Any | _T: ...
```

which i guess is equivalent to

```python
    def setdefault(self, key: str, failobj: ANy | None = None) -> Any: ...
```

right?

might as well remove the overloads?

Reply:
It's not equivalent. `.setdefault("x", None)` under my suggestion would return `Any | None`, which is not the same as `Any`.